pkg/virt-launcher/virtwrap/api/converter.go

```diff
@@ -1429,8 +1429,8 @@ func calculateRequestedVCPUs(cpuTopology *CPUTopology) uint32 {
 	return cpuTopology.Cores * cpuTopology.Sockets * cpuTopology.Threads
 }
 
-func CalculateNetworkQueueNumberAndGetCPUTopology(vmi *v1.VirtualMachineInstance) (uint32, *CPUTopology) {
-	cpuTopology := getCPUTopology(vmi)
+func CalculateNetworkQueueNumberAndGetCPUTopology(vmiSpec *v1.VirtualMachineInstanceSpec) (uint32, *CPUTopology) {
```

Comment:
I understand this workaround, but I just don't like that we have to do this.
I'll push another PR that removes the code that changes guests' topology based on the requested CPUs.